astro.config.mjs

```diff
@@ -19,12 +21,53 @@ import tqlLang from './tql.tmLanguage.json' assert { type: 'json' };
 const runLinkCheck = process.env.RUN_LINK_CHECK || false;
 const isProd = process.env.NODE_ENV === 'production';
 
+// Function to recursively find all .md files in a directory
```

Comment:
Can you move this into a separate file? This file should ideally mostly be config, not utility code.

Reply:
Done in 0dbfb7d5a454f3c435017b46419057fb304f0408

Reply:
I did go a bit further and moved all of the redirect logic out, that was easier.

Reply:
Good and bad. Now someone reading the configuration doesn't see the static redirects anymore, as they are buried in a utility function. I liked the previous approach with a mix of the two better, as it's quite easy to extend.

Reply:
My top priority is making it easy to extend/maintain. If I need to look deep into a library function...uh. It's bad enough that the top-level config file is code. But we don't have to make it worse.

Reply:
Right, and now if I want to add a redirect I can go to the "redirects" module and see all redirects in one place. Having them split across two would be confusing.